teuthology/run_tasks.py

```diff
@@ -25,6 +25,9 @@ def get_task(name):
     module = _import('teuthology.task', module_name, task_name)
     # If it is not found, try qa/ directory (if it is in sys.path)
     if not module:
+        module = _import('tasks', module_name, task_name)
+    if not module:
+        sys.path.append(ctx.config.get("suite_path") + "/qa")
```

Comment:
the suite_path setup by teuthology has qa, ex: http://qa-proxy.ceph.com/teuthology/teuthology-2019-12-18_07:00:04-smoke-nautilus-testing-basic-smithi/4613617/teuthology.log , shouldn't you do the same when running locally with custom qa path?

Reply:
If use teuthology-suite to submit some jobs,  will have change to use --suite-relpath parameter.  But, when i use teuthology-schedule to submit job,  seem unable to use relpath. 

i try to print suite_path in  https://github.com/ceph/teuthology/blob/master/teuthology/worker.py#L165 , and the result of suite_path is /home/teuthworker/src/github.com_ceph_ceph_xxx/.  I don't sure if there are other method to custom qa path. thanks